### ISSUE TITLE
chashing

### ISSUE THREAD

**Issue by CoolNateJr** [AntiCope/meteor-rejects#354]
### Describe the bug

it crashes when i join any worlds and servers

### Crashlog

[crash-2024-05-15_13.46.09-client.txt](https://github.com/AntiCope/meteor-rejects/files/15380759/crash-2024-05-15_13.46.09-client.txt)

### Steps to reproduce

1. joining a server or wolrd
2.
3.
...

### Operating System

None

### Meteor Version

0.5.7

### Minecraft Version

1.20.6

### Other mods

```plaintext
1.alot cant name them all but some other meteor plugins  like blackout and hig way tools and those kind of things
2.
3.
...
```

**Comment by ashton727** [comment]
Yeah same here, except I have nothing else but meteor and this mod and I crash as soon as I try to join a server

**Comment by crazymoose77756** [comment]
> Yeah same here, except I have nothing else but meteor and this mod and I crash as soon as I try to join a server

can you provide your crash log

**Comment by Risinget** [comment]
same, doesn't work with https://github.com/AntiCope/meteor-rejects/releases/download/latest-1.20.4/meteor-rejects-addon-0.3.jar with fabric 1.20.4 and meteor for 1.20.4 it crash... literally only i think this works 1.20.6 meteor rejects with latest dev build meteor

**Comment by crazymoose77756** [comment]
can you provide your crash log

**Comment by crazymoose77756** [comment]
> same, doesn't work with https://github.com/AntiCope/meteor-rejects/releases/download/latest-1.20.4/meteor-rejects-addon-0.3.jar with fabric 1.20.4 and meteor for 1.20.4 it crash... literally only i think this works 1.20.6 meteor rejects with latest dev build meteor

can you provide your crash log

**Comment by Risinget** [comment]
> > same, doesn't work with https://github.com/AntiCope/meteor-rejects/releases/download/latest-1.20.4/meteor-rejects-addon-0.3.jar with fabric 1.20.4 and meteor for 1.20.4 it crash... literally only i think this works 1.20.6 meteor rejects with latest dev build meteor
> 
> can you provide your crash log

[crash.log](https://github.com/user-attachments/files/15687672/crash.log)
